includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -830,14 +830,21 @@ public function is_missing_mandatory_attribute( $attr_spec, DOMElement $node ) {
 	 */
 	private function get_missing_mandatory_attributes( $attr_spec, DOMElement $node ) {
 		$missing_attributes = [];
+
 		foreach ( $attr_spec as $attr_name => $attr_spec_rule_value ) {
+			if ( ! isset( $attr_spec_rule_value[ AMP_Rule_Spec::MANDATORY ] ) || ! $attr_spec_rule_value[ AMP_Rule_Spec::MANDATORY ] ) {
```

Comment:
Can this be further optimized as:

```suggestion
			if ( empty( $attr_spec_rule_value[ AMP_Rule_Spec::MANDATORY ] ) ) {
```